x-pack/plugins/endpoint/server/config.ts

```diff
@@ -11,12 +11,17 @@ export type EndpointConfigType = ReturnType<typeof createConfig$> extends Observ
   ? P
   : ReturnType<typeof createConfig$>;
 
+const defaultPageSizeSchema = schema.number({ defaultValue: 10 });
```

Comment:
What's the reason for having this be configurable?

Reply:
> What's the reason for having 'default page size' configurable?
Yeah good point, I was more just following the crowd on this one haha, @madirey @nnamdifrankie thoughts?

> And what's the reason for having 'default first page' be variable?
I moved it to a variable because it seemed easier to me to type `defaultPageSizeSchema` rather than `schema.number({ defaultValue: 10 });` and since we were duplicating the same line a bunch of times.

Reply:
@jonathan-buttner how will the constant be called? on the instance or the class? We made it configurable on the page size, so that the default can be increased on the kibana instance. If desirable. But it is important to have all the config properties in one place, since we passed it through the app context.